src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -3858,6 +3826,26 @@ && isDlrmSetAsideEnabled(asylumCase)
         );
     }
 
+    @Bean
+    public PreSubmitCallbackHandler<AsylumCase> updateTribunalDecisionRule31NotificationHandler(
+            @Qualifier("updateTribunalDecisionRule31NotificationGenerator")
+            List<NotificationGenerator> notificationGenerators) {
+
+        return new NotificationHandler(
+                (callbackStage, callback) -> {
+                    AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
+
+                    return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
+                            && callback.getEvent() == Event.UPDATE_TRIBUNAL_DECISION
+                            && isDlrmSetAsideEnabled(asylumCase)
```

Comment:
Don't need this check, as the event is new.
Can you also add !AIP journey? Make sure we don't send this during AIP journey.